website/pages/contributors.html

```
@@ -11,6 +11,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Contributors | 100 Days of Web Dev</title>
     <link rel="stylesheet" href="../styles/index.css" />
+   <link rel="stylesheet" href="../../scripts/components/global.css">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ link ] must be self closed.

(tag-self-close)